src/features/import/utils/prepareImportOperations.ts

```diff
@@ -52,22 +86,26 @@ export default function prepareImportOperations(
               if (!personImportOps[rowIndex].tags) {
                 personImportOps[rowIndex].tags = [];
               }
-              personImportOps[rowIndex].tags = [
-                ...new Set(
-                  personImportOps[rowIndex].tags?.concat(mappedColumn.tagIds)
-                ),
+              const allTags = personImportOps[rowIndex].tags?.concat(
+                mappedColumn.tags.map((t) => ({ tag_id: t.id }))
+              );
+              const stringifiedUniqueTags = [
+                ...new Set(allTags?.map((t) => JSON.stringify(t))),
```

Comment:
OOoh I did not know about this quirk! Then it makes more sense to compare both properties!

Reply:
You know what I realise now! This was made when we had value tags, and now we don't have value tags in the import anymore, so this can be way simplified!